google/resource_compute_instance_group_manager.go

```diff
@@ -732,6 +817,42 @@ func expandAutoHealingPolicies(configured []interface{}) []*computeBeta.Instance
 	return autoHealingPolicies
 }
 
+func expandUpdatePolicy(configured []interface{}) *computeBeta.InstanceGroupManagerUpdatePolicy {
+	updatePolicy := &computeBeta.InstanceGroupManagerUpdatePolicy{}
+
+	for _, raw := range configured {
+		data := raw.(map[string]interface{})
+
+		updatePolicy.MinimalAction = data["minimal_action"].(string)
+		updatePolicy.Type = data["type"].(string)
+		updatePolicy.MaxSurge = &computeBeta.FixedOrPercent{
+			Fixed: int64(data["max_surge_fixed"].(int)),
+		}
+		updatePolicy.MaxUnavailable = &computeBeta.FixedOrPercent{
+			Fixed: int64(data["max_unavailable_fixed"].(int)),
+		}
+
+		// percent and fixed values are conflicting
+		// when the percent values are set, the fixed values will be ignored
+		if v := data["max_surge_percent"]; v.(int) > 0 {
```

Comment:
I'm wondering if we can move this up and then only set "Fixed" value in "else" block

Reply:
Definitely better in an else statement from readability perspective.